x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -30,29 +33,33 @@ HIPAA schemas also include details of code sets defined for elements. Data is va
 To use the X12 Module connector for HIPAA transaction sets you must:
 
 * Select one or more HIPAA schemas to be used.
-* Set the Form and Validation configuration parameter (`formValidation`) to 'HIPAA_SNIP1' or 'HIPAA_SNIP2', depending on the type of desired SNIP validation.
+
+* Set the Form and Validation configuration parameter (`formValidation`) to `HIPAA_SNIP1` or `HIPAA_SNIP2`, depending on the type of desired SNIP validation.
 
 === Determining HIPAA Schema Locations
 
 To use the connector, you need to know the locations of the schemas in your project.
-If you're using the out of the box HIPAA schemas and not customizing anything, the schema location follows the  `/hipaa/{version}/{transaction-set}.esl` convention.
-For example, if you're using the `005010X222A2` version and the 837 transaction set, your schema location is `/hipaa/005010X222A2/837.esl`.
+If you're using the out-of-the-box HIPAA schemas and not customizing anything, the schema location follows the  `/hipaa/{version}/{transaction-set}.esl` convention.
+For example, if you're using the `005010X222A2` version and the `837` transaction set, your schema location is `/hipaa/005010X222A2/837.esl`.
 
 If you’re using one or more custom schemas, place these under `src/main/resources` and refer to the location relative to this directory.
-For example, if you've put your HIPAA 837 schema under `src/main/app/mypartner/837.esl`, your schema location is `/mypartner/837.esl`.
+For example, if you've put your HIPAA `837` schema under `src/main/app/mypartner/837.esl`, your schema location is `/mypartner/837.esl`.
 
 == About HIPAA Schema Definitions
 
-The X12 Module uses a YAML format called ESL (for EDI Schema Language) to represent both standard X12 and HIPAA schemas.
+The X12 Module uses a YAML format called EDI Schema Language (ESL) to represent both standard X12 and HIPAA schemas.
 The HIPAA standards are based on X12, but modify the corresponding X12 base standards in several aspects including:
 
-* Multiple specialized definitions of HL (Hierarchical Level) loops, where a basic loop structure defined in X12 is expanded into several variations, often nested.
-* Modular loop definitions that can be reused at different points in the definition.
-* Distinguishing between different uses of a repeated segment by code values in a particular element.
-* Changing usage requirements for segments and segment components.
+* Multiple specialized definitions of Hierarchical Level (HL) loops, where a basic loop structure defined in X12 is expanded into several variations. These are often nested
```

Comment:
This sentence is missing something. It is not clear.

Reply:
REworded.